domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -87,4 +85,9 @@ object InteractionObjectTestBuilder {
   fun createInt(value: Int): InteractionObject {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
+
+  fun createListOf(vararg items: String): InteractionObject {
+    return InteractionObject.newBuilder().setListOf(items.toString()).build()
+  }
+
```

Comment:
This extra space is causing the lints checks to fail.

Reply:
Also, from our ktlint wiki page, make sure to setUp macros as per mention in the wiki page.